server/datastore/mysql/scripts.go

```diff
@@ -599,9 +599,16 @@ func (ds *Datastore) GetHostLockWipeStatus(ctx context.Context, hostID uint, fle
 			if err != nil {
 				return nil, ctxerr.Wrap(ctx, err, "get lock reference MDM command result")
 			}
-			// if there is are results, which one do we care about? Look for a failed
-			// or acknowledged one, the others probably say nothing about the result
-			// of the command (e.g. not now, pending)
+			// TODO: each item in the slice returned by
+			// GetMDMAppleCommandResults is a result for a
+			// different host. This only works because we're
+			// enqueuing the command with the given UUID for a
+			// single host, but it's the equivalent of doing
+			// cmdRes[0].
+			//
+			// Ideally, and to be super safe, we should try to find
+			// a command with a matching r.HostUUID, but we don't
+			// have the host UUID available.
```

Comment:
To be clear, that should be ok for now because the Lock MDM command is always for a single host, right? Later on we can improve this by e.g. passing the `*fleet.Host` instead of `hostID` and then we can check that the `HostUUID` matches in the list of responses?

Reply:
exactly! that what I was trying to convey but you put it way more nicely. I will update with your clarification post-freeze (or maybe we do the refactor you suggest?)